### ISSUE TITLE
Is there an official release for PRISMA_SCHEMA_ENGINE_BINARY and PRISMA_QUERY_ENGINE_LIBRARY?

### ISSUE THREAD

**Issue by KaiQiu9527** [prisma/prisma-engines#4447]
Hi all!
Our company is building all the projects with CI in a offline environment. 
When we build the prisma project, it cannot download PRISMA_SCHEMA_ENGINE_BINARY and PRISMA_QUERY_ENGINE_LIBRARY from internet.
According to the instructions, I can manually upload the two files and set the env params to use it locally. However, the two binary files are downloaded from the urls provided in the error message. Due to the company security policy, all the opensource binary files should be imported from an specific repo in github with proper license.
Is there a repo which providing all the RISMA_SCHEMA_ENGINE_BINARY and PRISMA_QUERY_ENGINE_LIBRARY binary files for being downloaded from?

**Comment by janpio** [comment]
No, that does not exist right now. These are only hosted on the official CDN the CLI downloads from.
There is a direct correlation of the engine repository commit hash, and the URL on the CDN.

(You can of course build them yourself from the source code of the respective git commit hash.)

I don't think it would be appropriate of us to use GitHub to host these binaries and use that as a distribution mechanism. It would also not be very fast and efficient, an actual CDN does a much better job at that.

What does your company need exactly to fulfil your internal guidelines?
